app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -357,6 +388,56 @@ class HtmlEditorState extends State<HtmlEditor> {
                 .bodySmall
                 .expect('bodySmall style not available'),
           ),
+      mobileDragHandleBallSize: const Size(12, 12),
+      textSpanDecorator:
+          widget.roomId != null ? customizeAttributeDecorator : null,
+    );
+  }
+
+  InlineSpan customizeAttributeDecorator(
```

Comment:
For customizing the styling of custom component block, here we can extend it beyond mentions (for future reference).